packages/webcert/src/feature/certificate/Inputs/GroupWrappers.tsx

```diff
@@ -0,0 +1,44 @@
+import { ConfigLayout } from '@frontend/common/'
+import styled, { css } from 'styled-components'
+
+interface checkboxesProps {
```

Comment:
Minor thing, but this should probably be named `GroupWrapperProps` and `checkboxProps` should be  `ItemWrapperProps`